sonar-ws/src/main/protobuf/ws-measures.proto

```diff
@@ -42,6 +42,11 @@ message ComponentWsResponse {
   optional Periods periods = 3;
 }
 
+// WS api/measures/search
+message SearchWsResponse {
+  repeated Component components = 1;
```

Comment:
measures will be returned at the 1st level, components will be available in another field